helm/values-dev.yaml

```
@@ -6,7 +6,15 @@ service:
     metricsMonitor:
       enable: false
     healthCheck:
-      enable: false
+      enable: "true"
+      liveness: 
+        initialDelaySeconds: 15
+        periodSeconds: 5
```

```suggestion
        periodSeconds: 5
        timeoutSeconds: 1
        failureThreshold: 3
        successThreshold: 1
```

Let's be explicit about these settings so there is no confusion as to what each one is configured with.  The values I've set for `timeout`, `failure`, and `success` are the defaults kubernetes would set.